metecho/api/models.py

```diff
@@ -80,6 +81,10 @@ class User(HashIdMixin, AbstractUser):
     def notify(self, subject, body):
         # Right now, the only way we notify is via email. In future, we
         # may add in-app notifications.
+
+        # Escape <>& in case the email gets accidentally rendered as HTML
+        subject = html.escape(subject)
+        body = html.escape(body)
```

Comment:
@davisagli @wlonk This is escaping too aggressively -- any way we can allow quotes and apostrophes?

![Screenshot 2020-07-01 16 33 51](https://user-images.githubusercontent.com/552316/86288931-c6171700-bbb8-11ea-929c-0bf8bf2b1f17.png)

Reply:
Oops -- I intended to. html.escape takes a boolean `quote` argument but I made the wrong assumption about its default value.

Reply:
@davisagli Got it; no prob. Fixed in https://github.com/oddbird/Metecho/commit/2674936476e2551f571a3b703115b002843f4341